src/tests/src/expressions/test_PrintAndEvalVisitors.cpp

```diff
@@ -824,26 +823,24 @@ BOOST_FIXTURE_TEST_CASE(evaluate_shifted_literal, MyDummyFixture)
       EvaluationResult::EvalResultTypeError);
 }
 
-template<typename left, typename right>
-EvaluationResult CreateAndEvaluateTimeNode(const right& p)
+template<typename NodeType>
+EvaluationResult CreateAndEvaluateTimeNode(Node* p)
 {
     ParameterNode paramNode("my-param", VariabilityType::VARYING_IN_TIME_ONLY);
-    left root(&paramNode, p);
+    NodeType root(&paramNode, p);
     const std::string value = "dummy";
     MockLinearProblemData dummy_data;
     Antares::Optimisation::LinearProblemApi::EmptyScenario emptyScenario;
 
     unsigned first = 0;
     unsigned last = 2;
 
-    const auto param_type = ParameterType::TIMESERIE;
     Model model = createModelWithParameters(
       {Parameter("my-param", TimeDependent::YES, ScenarioDependent::NO)});
-    auto param = build_context_parameter_with("my-param", value, param_type);
+    auto param = build_context_parameter_with("my-param", value, ParameterType::TIMESERIE);
```

Comment:
Even in singular, timeseries has an "s" in english
```suggestion
    auto param = build_context_parameter_with("my-param", value, ParameterType::TIMESERIES);
```

Reply:
Done